### PR TITLE
container: update inference dockerfile for new model design

### PR DESCRIPTION
**Pull request by dongx1x**
- Remove downloading model to docker image, models will be downloaded from model server when deploying inference module.
- Update helm charts default env to support new model design, the model informations will be obtained from model info server.

### CODE REVIEW

File: container/cnap-inference/Dockerfile [hunk 1]
```diff
@@ -30,11 +12,7 @@ ARG pip_mirror
 
 COPY ./cnap /cnap
 
-COPY --from=downloader \
-    /public/ssd_mobilenet_v1_coco/ssd_mobilenet_v1_coco_2018_01_28/frozen_inference_graph.pb \
-    /demo/model/model.pb
-
-RUN chown -R appuser:appuser /demo /cnap
+RUN chown -R appuser:appuser /cnap
```
Comment (dongx1x) [inline container/cnap-inference/Dockerfile:15]: @Hulongyin I'm not sure about removing the `/demo` directory, please help confirm this change.

Comment (Hulongyin) [reply container/cnap-inference/Dockerfile:15]: Ok, the `/demo` directory in inference service was used to store the models, but now it's not needed, so it's OK to remove it.

File: container/cnap-inference/Dockerfile
Comment (Hulongyin) [inline container/cnap-inference/Dockerfile:1]: The `ccnp` and `requests` should be installed.

Comment (dongx1x) [reply container/cnap-inference/Dockerfile:1]: Yes, updated. Thanks.

**Review by Hulongyin — APPROVED** [review]
LGTM. Thanks

**Pull request merged**